.env.example

```
@@ -10,6 +10,13 @@ AUTH_SECRET=           # openssl rand -hex 32  (JWT signing secret)
 
 NEXT_PUBLIC_API_BASE=  # e.g. https://api.yourdomain.com
 
+# REDIS_PASSWORD: issue #191/S3 -- auths the webhook ingestion queue (Redis
+# Streams). apps/api/entrypoint.sh builds REDIS_URL from this at container start
+# (same pattern as DB_USER/DB_PASSWORD/DB_NAME -> DATABASE_URL below). No safe
+# default: without it the redis service won't start (--requirepass) and the
+# webhook receiver has no queue to reach.
+REDIS_PASSWORD=  # openssl rand -hex 32
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)